api/catalog/api/serializers/media_serializers.py

```diff
@@ -107,22 +106,16 @@ class MediaSearchRequestSerializer(serializers.Serializer):
         required=False,
         default=False,
     )
-    page = serializers.IntegerField(
-        min_value=1,
-        default=1,
-        help_text="The index of the page of the results to show.",
-    )
     page_size = serializers.IntegerField(
-        min_value=1,
-        max_value=500,
-        default=20,
-        help_text="The number of results to show in one page.",
+        label="page_size",
+        help_text="Number of results to return per page.",
+        required=False,
```

Comment:
We should keep this validation here as is the most appropriate place, though now the validation on the `api/catalog/api/utils/pagination.py` file was re-added so it's not blocking this.